doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -698,8 +698,16 @@ Matter samples
   * WPA supplicant advanced features in all Matter samples using the :kconfig:option:`CONFIG_WPA_SUPP_ADVANCED_FEATURES` Kconfig option.
     This saves roughly 25 KB of FLASH memory for firmware images with Wi-Fi support.
 
-* Added ``matter_shell`` shell commands set to gather the current information about the NVS settings backend such as current usage, free space, and peak usage value.
-  You can enable them by setting the :kconfig:option:`NCS_SAMPLE_MATTER_SETTINGS_SHELL` Kconfig option to ``y``.
+* Added:
+  * ``matter_shell`` shell commands set to gather the current information about the NVS settings backend such as current usage, free space, and peak usage value.
+    You can enable them by setting the :kconfig:option:`NCS_SAMPLE_MATTER_SETTINGS_SHELL` Kconfig option to ``y``.
+  * :file:`pm_static_<BOARD>` Partition Manager configuration files for all required target boards to the samples's directory.
```

Comment:
Hmm, maybe it's better to include this point in "Updated" section and combine it with the first "Removed" bulled point?